docs/diagrams/instruction.md

```diff
@@ -0,0 +1,22 @@
+### Getting started
+
+- Install plantuml `cs install --contrib plantuml`
+- vscode has extension which allows to preview creating diagram - [PlantUML](https://marketplace.visualstudio.com/items?itemName=jebbs.plantuml)
+
+### Usage
+
+Documentation can be found at [website](https://plantuml.com/command-line). Following instruction is more or less based on mentioned website.
```

Comment:
```suggestion
Documentation can be found on the [website](https://plantuml.com/command-line).
```
You can probably leave the second sentence out since it's not really adding anything.